### ISSUE TITLE
[Bug]: Metamask infinite load making it inoperative.

### ISSUE THREAD

**Issue by tdev00** [MetaMask/metamask-extension#21874]
### Describe the bug

MetaMask displays an infinite loading spinner without end.

![image](https://github.com/MetaMask/metamask-extension/assets/63981572/6d1d2484-c07d-4879-9f36-e70d47942c6c)

### Expected behavior

MetaMask should display all the wallets after entering credentials.

### Screenshots/Recordings

![image](https://github.com/MetaMask/metamask-extension/assets/63981572/4e9bc01a-5253-4f8c-ba48-a8f0409ec062)
![image](https://github.com/MetaMask/metamask-extension/assets/63981572/4e3c2a33-7323-4a74-9f74-8f28f26d6d67)

### Steps to reproduce

I realize that version 11.5.0 has been released. The issue still persists even after updating the extension.

Here are the steps I've taken:

- Cleared the browser cache and cookies.
- Closed the browser completely and restarted the PC.
- Disabled the VPN.
- Tried to remove the loading overlay via CSS to access the settings.
- Ensured the extension's access is set to "On all sites."
- Attempted to open the extension in incognito mode.
- Updated Chrome. Current version: 119.0.6045.159 (Official Build) (64-bit)
- Updated MM Chrome extension to 11.5.0.

_**N.B.** Issue does not appear on Brave browser_

### Error messages or log output

```shell
Error#225: PollingBlockTracker - encountered an error while attempting to update latest block:
{
  "code": -32603,
  "message": "Internal JSON-RPC error.",
  "data": {
    "code": -32010,
    "message": "Public RPC endpoints at gateway.pokt.network are deprecated as of November 1 2023. For continued service please use Pocket's new public endpoints at https://docs.pokt.network/use/public-rpc."
  },
  "stack": "Error: Internal JSON-RPC error.\n    at new i (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/common-3.js:16:10940)\n    at s (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/common-3.js:16:13880)\n    at Object.internal (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/common-3.js:16:14490)\n    at c (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background-0.js:7:44976)\n    at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background-0.js:7:45978\n    at async chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background-4.js:12:53970"
}
  at new i (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/common-3.js:16:10940)
  at s (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/common-3.js:16:13880)
  at Object.internal (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/common-3.js:16:14490)
  at c (chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background-0.js:7:44976)
  at chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/background-0.js:7:45978
```

### Version

11.5.0

### Build type

None

### Browser

Chrome

### Operating system

Windows

### Additional context

A ticket related to this issue has been closed as version 11.5.0 hotfix was released. The issue has not been fixed.

Related tickets: 

1. https://github.com/MetaMask/metamask-extension/issues/21598
2. https://github.com/MetaMask/metamask-extension/issues/21719

**Comment by anaamolnar** [comment]
Hello, @tdev00. Thanks for reporting! The team is actively working on deploying a fix for this issue. We will be releasing it with the next version, 11.5.1. Thank you for your patience and understanding!

**Comment by danjm** [comment]
This was fixed in https://github.com/MetaMask/metamask-extension/pull/21854, which will be in v11.5.1

v11.5.1 was submitted to chrome yesterday, so hopefully it will be available to users some time today. I will comment here again once it is available to users

**Comment by harleauxcarrera** [comment]
Thanks for reporting and thanks for the support @anaamolnar @danjm. 
I'm sure you can understand how time sensitive this bug should be prioritized as since it is rendering the application useless...

not a good feeling after having just moved over significant amount of capital.... 
not sleeping until this is fixed

**Comment by carynx** [comment]
> Hello, @tdev00. Thanks for reporting! The team is actively working on deploying a fix for this issue. We will be releasing it with the next version, 11.5.1. Thank you for your patience and understanding!


Hi, I have this issue too. metamask does not load up.  I have version 11.4.1, using firefox. 
what makes me freak out is that I just now understood that my accounts within my wallet also have private keys, I didnt know that..  so therefore i cant just remove metamask and reinstall it.  what should i do? wait for update and pray?

**Comment by carynx** [comment]
i right click on the metasmask extension, and click manage extension, then i click settings symbol upper right, and then check for updates, no updates found  

![MMISSUE](https://github.com/MetaMask/metamask-extension/assets/151438827/16e0d540-f01b-4a70-b9c9-dae27bc74786)

**Comment by carynx** [comment]
no updates found,  and i dont know how to update in manually using the downloadable files you have provided, any guide to that.  I just dont want to reinstall my wallet, because i dont have the private key to one of the accounts within the wallet. 

![MMISSUE](https://github.com/MetaMask/metamask-extension/assets/151438827/55c085dc-f02b-4de1-97f3-4a526109ee9f)

**Comment by sandakersmann** [comment]
I have the same issue on Firefox.

**Comment by danjm** [comment]
v11.5.1 has been released on chrome. Users experiencing this issue on chrome can manually update to v11.5.1

Here are instructions on how to do that https://support.metamask.io/hc/en-us/articles/360060268452-How-to-update-the-version-of-MetaMask

**Comment by sandakersmann** [comment]
@danjm When will the Firefox extension be updated?

**Comment by freeappmm** [comment]
Same here for Firefox.

**Comment by sandakersmann** [comment]
@Twente-Mining I had to reinstall the extension.

**Comment by Twente-Mining** [comment]
> @Twente-Mining I had to reinstall the extension.

Thanks, but that is not really a fix and will remove the private keys loaded.
a lot of hassle to get them back in.

**Comment by yaerna1984** [comment]
Same thing happened to me on Android. 😢

**Comment by gauthierpetetin** [comment]
Duplicate: https://github.com/MetaMask/metamask-extension/issues/24777

**Comment by gauthierpetetin** [comment]
Duplicate: https://github.com/MetaMask/metamask-extension/issues/24782

**Comment by gauthierpetetin** [comment]
The cause of this problem related to network connection was fixed in 11.5.1.
If you're still facing a similar issue, this issue is the possible cause: https://github.com/MetaMask/metamask-extension/issues/9196
